services/navigation/builtin/builtin_test.go

```diff
@@ -293,10 +295,16 @@ func TestNew(t *testing.T) {
 			PositionPollingFrequencyHz: 3,
 			ObstaclePollingFrequencyHz: 4,
 			PlanDeviationM:             5,
-			VisionServices:             []string{"vision"},
+			ObstacleDetectors: []*motion.ObstacleDetectorConfig{
```

Comment:
[q/opt] Do we want to add test cases for if base, motion, movement_sensor, camera, vision service are unable to be gotten `FromDependencies`?

Reply:
I can add the missing cases